recipe/provision/website.php

```diff
@@ -68,9 +66,8 @@
         run("echo $'$caddyfile' > Caddyfile");
     }
 
-    set('remote_user', 'root');
     if (!test("grep -q 'import $deployPath/Caddyfile' /etc/caddy/Caddyfile")) {
-        run("echo 'import $deployPath/Caddyfile' >> /etc/caddy/Caddyfile");
+        run("echo 'import $deployPath/Caddyfile' | sudo tee -a /etc/caddy/Caddyfile");
```

Comment:
We can't use this as this will require a password on some hosts.

Reply:
I chose to do it like that because I found the same concept here: https://github.com/deployphp/deployer/blob/68a983a8ca52e6ffd44ad08f139d3dc94fb570da/recipe/provision.php#L94

It would be consistent with the other code.

Reply:
I removed "sudo" since it works without it when using `-o become=root`.